common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1120,6 +1120,15 @@ def _block_matches_all(block_data):
         else:
             return []
 
+    def has_path_to_root(self, xblock, course):
```

Comment:
I don't see why we need this assert; I just thought `block_key` was a more descriptive name than `xblock`